src/components/component-card/ComponentCard.css

```diff
@@ -11,6 +11,7 @@
   background-position: center;
   border-bottom: 1px solid var(--goa-color-greyscale-200);
   background-color: #F8F8F8;
+  border-radius: 3px 3px 0px 0px;
```

Comment:
When setting border-radius here, it can result in unwanted pixels resulted due to the anti-aliasing of the two borders. A better way of achieving what you want to do is to set `overlap: hidden` on the parent elements css (in this case the `.card`)

Reply:
Good to know, thanks. updated.

Reply:
@chrisolsen Wasn't working, but used overflow: hidden instead. Updated again, looks to be working now. Good to review.

Reply:
@twjeffery That is what I meant. Good catch :)